### DISCUSSION TITLE
I don't want to lose the GoPro password when reset - what can I do?

### DISCUSSION THREAD

**Discussion by mactac** [gopro/labs#284] [Q&A]
A bit of background:

I have a few (6) GoPro session 5 cameras that have dead LCD displays. They still function, and most importantly, I still have access to them via the app so I can change settings, etc.

Here's the problem: If I ever have to re-install the app, it won't be able to connect without the password which I can't get because there is no display. If I ever have to reset my camera, it'll also change the password, so I won't be able to connect anymore.

Is there anything that I can do while I still have access to the cameras via the app to deal with this situation perhaps with GoPro Labs? Like lock the password to a specific password, or remove the wifi password altogether or something?

Failing that, perhaps there is something that can be done in CLI/bootup commands?

Any ideas would be appreciated, I don't want to eventually brick the cameras because I don't have the wifi password or if it changes.

**Comment by dnewman-gpsw** [comment]
I believe this is a solution, but it is not simple without a screen. Before you change anything else, make sure the camera default mode is video (likely is.)

1) If Labs is not installed, install Labs for HERO5 Session  https://community.gopro.com/s/article/GoPro-Labs?language=en_US
2) Boot the camera via the mode button
3) 5-8 seconds later press mode again (this will be in the menu)
4) The camera is ready to scan QR Code.
5) Now you can set a new SSID and new PASSWORD with the command oWANewSSID,Password  like oWACAMERA_A,PASS1234  (only use uppercase or numbers or _, it will not work others, and at least eight characters for each.)
6) After the scan, quickly press the shutter button 3 times (this is the approve request.)
7) Then scan !PA - Pairing App
8) After the scan, quickly press the shutter button 3 times again (this is the approve pairing request.)
9) Then the App will content with the camera and use the new SSID and Password.

It should all work with new SSID and password of your choice.
